performance/performance_tuning.rst

```diff
@@ -44,10 +44,10 @@ Here is an example of explaining the plan for a particular example query.
 ::
 
   Sort  (cost=0.00..0.00 rows=0 width=0)
-    Sort Key: minute
+    Sort Key: remote_scan.minute
```

Comment:
These are fairly mechanical changes, but I'd like your suggestions about what broader advice to give. The "Advanced" section is especially lacking.